script/1_0_2current_en_script/Ciel/Day 3/03_01_CIEL03_1.txt

```diff
@@ -1,30 +1,30 @@
 <Page1>
 When I enter the classroom, I find that homeroom is about to start.
 I wasn't used to the route, so I must have paced myself wrong.
-The morning chatting period is coming to an end, and most of the students have gone back to their desks.
+The morning chatter is starting to dwindle, and most of the students have gone back to their desks.
 <Page2>
 In that situation,
-"...Hah, you've arrived with a lot of time to spare, huh?"
-Arihiko is standing there with a dejected look on his face, about to take his seat. ...He seems to be taking class seriously today.
+"...Hah, you've sure arrived with a lot of time to spare, huh?"
+Arihiko is standing there with a dejected look on his face, about to take his seat. ...He actually seems to be taking class seriously today.
 <Page3>
-"Good morning. Not too energetic this morning I see...# no that can't be it. You came to the classroom after all. What? Are you feeling unwell?"
+"Good morning. Not too energetic this morning I see...# No that can't be it. You made it to the classroom after all. What is it? You feeling unwell?"
 <Page4>
-"I was full of energy until a while ago. After all, Senpai came to visit me."
-"By Senpai, you mean Ciel-senpai?"
+"I was brimming with energy until a while ago. After all, Senpai came over to visit."
+"By Senpai, do you mean Ciel-senpai?"
 <Page5>
 "That's right. Buuut, given a certain someone wasn't here, she didn't quite seem to be in the mood anymore and went back to her classroom. Why's it that this, of all mornings, is when you decide to show up at the very last minute?"
 <Page6>
 So that's what happened... if I knew that Senpai was waiting, I'd have come here running.
 <Page7>
-"Sorry about that. But there's always the possibility we can meet up with her at lunch. It's not like this was our last chance to see her."
+"Sorry about that. But there's always the possibility we can meet up with her at lunch. It's not like this was our last chance."
```

Comment:
there's always the possibility that we can meet up with her at lunch. It's not like that was our last chance.